docs/03-language-guide/14-equality.md

```diff
@@ -1,6 +1,131 @@
 ---
 title: Equality
 id: equality
+description: A reference guide to equality in Wing.
+keywords: [Winglang equality, Wing equality, equals, identity]
```

Comment:
I don't think we need to include `wing` in our keywords. Ideally, we should find a way to inject it for every page in the docs automatically. or maybe it's just not needed at all.

```suggestion
keywords: [equality, equals, identity]
```